Emit extensions

pv42 opened the pull request:
Fixes emit-extensions and emit-descriptions features not being used by mavlink-bindgen (#246).
This requires a (none breaking) bump in version since fixes in the mavlink crate require newer versions of mavlink-bindgen/mavlink-core crates.
To ensure test work this required adding the path specifications for the dependencies within this crate.

Fixes missing storm32 dialect feature.

patrickelectric commented on mavlink/Cargo.toml:
```diff
@@ -19,15 +19,15 @@ edition = "2018"
 required-features = ["ardupilotmega"]
 
 [dependencies]
-mavlink-core = { version = "0.13.1", default-features = false }
```
You should avoid the following for the release system to work.

pv42 replied:
I have removed the versions in 33b4d4ec3c971bb1af81080c31a77304a790096e but this breaks most tests.
Looking at the failed build log of 9cd7bf0b8c720d255232a389051fb60f5fc92c2c before the path dependencies where removed the issue seems to be that no version at all is specified:
> error: all dependencies must have a version specified when publishing.

But since cargo supports specifying a [path and a version](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#multiple-locations) doing so should circumvent the issue.

pv42 replied:
resolved based with merged PR #255 that revert to using paths

amsmith-pro commented:
Looking forward to this as well, thank you

pv42 commented:
I was originally under the impression storm32 was just overlocked and since it generated a compiler warning I just added it.
But #165 indicated it was intentionally left out. I also probably should never have been in this unrelated PR. Therefore I removed it.

patrickelectric reviewed (approved):
Thanks! After brainstorming more we decided to continue with the path approach and fix the release method in a different way.

patrickelectric commented:
@pv42 can you rebase over master ? It appears to contain conflicts.

pv42 commented:
Fixed the conflicts with 0fb0e2e.

pv42 commented:
Should have be properly rebased now.
